xrpl/asyncio/ledger/main.py

```diff
@@ -54,13 +55,26 @@ async def get_fee(client: Client) -> str:
         client: the network client used to make network calls.
 
     Returns:
-        The minimum fee for transactions.
+        The minimum fee for transactions, in drops.
 
     Raises:
         XRPLRequestFailureException: if the rippled API call fails.
     """
-    response = await client.request_impl(Fee())
-    if response.is_successful():
-        return cast(str, response.result["drops"]["minimum_fee"])
+    response = await client.request_impl(ServerInfo())
+    if not response.is_successful():
+        raise XRPLRequestFailureException(response.result)
 
-    raise XRPLRequestFailureException(response.result)
+    server_info = response.result["info"]
+    if "validated_ledger" not in server_info:
+        raise XRPLRequestFailureException(response.result)
+
+    base_fee = int(xrp_to_drops(server_info["validated_ledger"]["base_fee_xrp"]))
+
+    if "load_factor" not in server_info:
+        load_factor = 1
+    else:
+        load_factor = server_info["load_factor"]
+
+    fee = base_fee * load_factor
+    # TODO: add cushion and max fee params to this method
```

Comment:
Max fee is super important. We _really_ don't want people to auto-fill a really high fee because they happened to catch a server at its busiest. People tend to get upset if sending a transaction burns 1000x as much as it delivers, and auto-filling doesn't always provide an opportunity to double-check that first.